packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -463,16 +465,28 @@ export function makeVatKeeper(
     }
   }
 
+  function transcriptSize() {
+    const bounds = transcriptStore.getCurrentSpanBounds(vatID);
+    const { startPos, endPos } = bounds;
+    return endPos - startPos;
+  }
+
   /**
-   * Generator function to return the vat's transcript, one entry at a time.
+   * Generator function to return the vat's current-span transcript,
+   * one entry at a time.
    *
-   * @param {number} [startPos]  Optional position to begin reading from
```

Comment:
Is it really good to lose the start position here?  Normally you only replay the current transcript span, but we at least anticipate scenarios where we need to go further back in time.